server/app.test.ts

```diff
@@ -0,0 +1,15 @@
+import request from 'supertest'
+import express from 'express'
+import app from './index'
```

Comment:
This is a very basic test and was the first one I wrote. To be honest it probably doesnt add much value in itself, especially with the other tests I wrote in this PR that test the same endpoint in more detail